mi/dataset/driver/ctdmo_ghqr/sio/resource/CTD02001_CT.yml

```diff
@@ -1,6 +1,6 @@
 header:
   particle_object: CtdmoGhqrRecoveredHostInstrumentDataParticle
-  particle_type: ctdmo_ghqr_instrument_host_recovered
+  particle_type: ctdmo_ghqr_instrument_recovered
```

Comment:
I thought when we researched the preloaded database we found the ctdmo_ghqr_sio_mule_instrument telemetered data stream to be what you needed.  Unless you investigated the preloaded database further and found ctdmo_ghqr_instrument_recovered to be closer.